src/apps/api/tests/test_queue_api_permissions.py

```diff
@@ -4,15 +4,15 @@
 from django.test import TestCase
 from django.urls import reverse
 
-from factories import UserFactory
+from factories import UserFactory, QueueFactory
 from queues.models import Queue
 
 
 class QueuesAPITestCase(TestCase):
     def setUp(self):
         self.user = UserFactory(username='test')
         self.another_user = UserFactory(username='another_user')
-        self.organizer_user = UserFactory(username='organizer_user')
+        self.collab_user = UserFactory(username='organizer_user')
         # Prevent queue creation
         with mock.patch('queues.models.rabbit.create_queue') as rabbit_create_queue:
```

Comment:
This should get abstracted to a helper function called something like `mock_create_queue` and it should use `QueueFactory()` and then use this function in place of `QueueFactory` so that anywhere below that you are currently using `QueueFactory`, `create_queue` will get mocked correctly every time. (that sentence is a mess... let me know if it doesn't make sense...)

Reply:
✔️